### DISCUSSION TITLE
This is broken.

### DISCUSSION THREAD

**Discussion by GameLord2011** [matteobruni/tsparticles#5413] [Q&A]
I am trying to use this in my website, and it is not working at all. The repo is [GameLord2011/GameLord2011.github.io](https://github.com/GameLord2011/GameLord2011.github.io)

**Comment by GameLord2011** [comment]
I know the code is working, it makes the canvas and evrything.

**Reply by matteobruni** [reply]
You are using the syntax of version 2.x on a 3.x script. Please check the readme of this repo.

**Comment by GameLord2011** [comment]
ok Im using npm now.

**Comment by GameLord2011** [comment]
Yea, I suck at coding and can't figure this out. Can you post the steps on
how to use this with node on here, cuz i've tried using it and can't.

**Reply by matteobruni** [reply]
Please check the readme first, the steps are all there

**Comment by GameLord2011** [comment]
I am using cdn.js and it is kinda working
-

**Comment by GameLord2011** [comment]
Can you just send me the correct code? cuz i am runnnin out of scripts &
tricks here

**Comment by GameLord2011** [comment]
I am trying to use it w/ npm because that is the "correct" way and I did everything the readme says, and it is saying that require is not defined when I try to load it.

**Reply by matteobruni** [reply]
https://github.com/GameLord2011/GameLord2011.github.io/blob/d75337f490187dffa0f5586ffa27b95c50ab2002/assets/js/load.js#L1

this is not something that is in the readme. The import is wrong, you can see the right imports here:

https://github.com/tsparticles/tsparticles/tree/main/bundles/all

**Reply by GameLord2011** [reply]
can you just commit the correct code w/ my options into the loading script and replace the package i already have w/ the correct one(s)?
